motor_controller/motorbench-test-project/motorbench-test-project/rb_library/rb_measure.c

```diff
@@ -1,6 +1,6 @@
 #include "rb_measure.h"
-#include "../motorBench/hal/hardware_access_functions.h"
-#include "../mcc_generated_files/adc/adc1.h"
+#include "hal/hardware_access_functions.h"
+#include "adc/adc1.h"
 
 
 void RB_ADCCompensationInit(RB_MEASURE_CURRENT_T *pcal)
```

Comment:
Slightly confusing naming. Is cal short for calibration? If so, what is the difference between calibration and compensation? If you decide to keep the names the same, maybe add a description of the function and parameters in the .h file

Reply:
cal is short for calibration.
Calibration is determining the offsets
Compensation is applying the offsets
pcal holds the calibration/compensation constants.

I added comments in the .h

Reply:
Ok that makes sense, compensation function requires the original calibration settings. But doesn't that mean this is the  _calibration_ init and the RB_MeasureCurrentOffsetStepISR is the adc calibration step?

Reply:
yeah